static/app/utils/eventWaiter.tsx

```diff
@@ -123,14 +136,22 @@ class EventWaiter extends Component<EventWaiterProps, EventWaiterState> {
         organization,
         project,
       });
-    } else {
-      firstIssue = firstEvent;
+    } else if (eventType === 'transaction') {
+      firstIssue = Boolean(firstEvent);
       // noinspection SpellCheckingInspection
       recordAnalyticsFirstEvent({
         key: 'first_transaction_recieved',
         organization,
         project,
       });
+    } else if (eventType === 'replay') {
+      firstIssue = Boolean(firstEvent);
+      // noinspection SpellCheckingInspection
```

Comment:
what's this? 👀

Reply:
Copy+paste from above!

also it's gone now.